config/generic_update.py

```diff
@@ -0,0 +1,332 @@
+import jsonpatch
+import sonic_yang
+import os
+load_source('sonic_cfggen', '/usr/local/bin/sonic-cfggen')
+from sonic_cfggen import deep_update, FormatConverter
+
+class JsonChange:
+    def __init__(self, patch):
+        self.patch = patch
+
+    def apply(self, current_json):
+        return self.patch.apply(current_json)
+
+    def __str__(self):
+        return f"{self.patch}"
+
+class ConfigLocker():
+    def acquire_lock(self):
+        # TODO: Implement ConfigLocker
+        pass
+
+    def release_lock(self):
+        # TODO: Implement ConfigLocker
+        pass
+
+class NoOpConfigLocker(ConfigLocker):
+    def acquire_lock(self):
+        pass
+    def release_lock(self):
+        pass
+
+class PatchOrderer:
+    def order(patch):
+        # TODO: Implement patch orderer
+        pass
+
+class ChangeApplier:
+    def apply(change):
+        # TODO: Implement change applier
+        pass
+
+YANG_DIR = "/usr/local/yang-models"
+class ConfigWrapper:
+    def get_current_config(self):
+        config_db_json = __get_config_db_as_json()
+        return __convert_config_db_to_yang(config_db_json)
+
+    def validate_config(self, yang_as_json):
+        config_db_as_json = __convert_yang_to_config_db(yang_as_json)
+        config_as_yang
+
+        sy = sonic_yang.SonicYang(YANG_DIR, debug=debug)
+        sy.loadYangModel()
+        sy.loadData(config_db_as_json)
+
+        try:
+            sy.validate_data_tree()
+            return True
+        except Exception as ex:
```

Comment:
>Exception [](start = 15, length = 9)

Do not catch general exception type unless you have a good reason.

Reply:
Code updated to catch specific exceptions,